tools/macos.py

```diff
@@ -11,6 +12,7 @@ def has_osxcross():
 def options(opts):
     opts.Add("macos_deployment_target", "macOS deployment target", "default")
     opts.Add("macos_sdk_path", "macOS SDK path", "")
+    opts.Add(BoolVariable("with_x86_64h", "If arch=x86_64 or universal, include an x86_64h slice.", False))
```

Comment:
"with_" as a prefix for this option doesn't really fit with the naming of our other options. Maybe "use_x86_64h"?